Resources/ServerInfo/Guidebook/Security/Defusal.xml

```diff
@@ -0,0 +1,73 @@
+
+  ## Time
+  A bomb has a limited time, at a minimum of 90 and a maximum of 300. You can view the timer by examining it, unless the Proceed wire is cut. Once the timer hits zero, the bomb will detonate.
+
+  ## Bolts
+  By default, once armed, a bomb will bolt itself to the ground. You must find the BOLT wire and cut it to disable the bolts, after which you can unwrench it and throw it into space.
+
+  ## Wires
+  You must access the wiring in order to defuse a bomb. You can use a [color=yellow]screwdriver[/color] to open the access panel. Inside, you will find many types of wires. In a standard syndicate bomb, there are around [color=yellow]10 wires[/color], 3 are dummy wires, [color=red]3 will cause a detonation[/color], and the rest that weren't mentioned can be found below (alongside BOOM wires). With each wire, you can do 3 actions. You can:
+  - [color=yellow]Pulse the wire[/color] with a multitool, this can help you safely identify most wires.
+  - [color=red]Cut the wire[/color] with a wirecutter, this can trigger various effects, be cautious of cutting without reason!
+  - [color=green]Mend the wire[/color] with a wirecutter, this can restore some functionality of the bomb if it isn't disposable.
+
+  Onward for the types of wires.
+
+  ## Wire Types
+  [color=#a4885c]Activation Wire (LIVE)[/color]
+  - [color=yellow]Pulse the wire[/color]: Pulsing the wire will make the wire chirp and delay the bomb by 30 seconds.
+  - [color=red]Cut the wire[/color]: Cutting the wire will defuse the bomb if active, otherwise, will begin the timer.
+  - [color=green]Mend the wire[/color]: Nothing.
+
+  [color=#a4885c]Proceed Wire (PRCD)[/color]
+  - [color=yellow]Pulse the wire[/color]: Pulsing the wire will forward the time by 15 seconds.
+  - [color=red]Cut the wire[/color]: Cutting the wire will disable the timer display on examine.
+  - [color=green]Mend the wire[/color]: Nothing.
+
+  [color=#a4885c]Delay Wire (DLAY)[/color]
+  - [color=yellow]Pulse the wire[/color]: Pulsing the delay wire will delay the bomb by 30 seconds.
+  - [color=red]Cut the wire[/color]: Nothing.
+  - [color=green]Mend the wire[/color]: Nothing.
+
+  [color=#a4885c]Boom Wire (BOOM)[/color]
+  - [color=yellow]Pulse the wire[/color]: [color=red]The bomb will explode if armed![/color]
+  - [color=red]Cut the wire[/color]: [color=red]The bomb will explode if armed![/color] Otherwise, will disable the bomb.
+  - [color=green]Mend the wire[/color]: Re-enables the bomb if disabled previously.
+
+  [color=#a4885c]Bolt Wire (BOLT)[/color]
+  - [color=yellow]Pulse the wire[/color]: Pulsing the wire will make the bolts spin.
+  - [color=red]Cut the wire[/color]: Cutting the wire will disable the bolts, throw it into space!
+  - [color=green]Mend the wire[/color]: Mending the wire will re-enable the bolts.
+
+  [color=#a4885c]Dummy Wire[/color]
+  Dummy wires don't do anything. You can pulse, cut, and mend them freely and they will not affect the bomb at all.
```

Comment:
```suggestion
- Dummy wires don't do anything. You can pulse, cut, and mend them freely and they will not affect the bomb at all.
```

doesnt have spacing ingame otherwise